resources/js/requests/components/RequestsListing.vue

```diff
@@ -13,7 +13,7 @@
         pagination-path="meta"
       >
         <template slot="ids" slot-scope="props">
-          <b-link @click="openRequest(props.rowData, props.rowIndex)">{{props.rowData.name}}</b-link>
+          <b-link @click="openRequest(props.rowData, props.rowIndex)">#{{props.rowData.id}}</b-link>
```

Comment:
@marcoAntonioNina this shouldn't replace the name column. We are adding a column at the beginning of the table.

Reply:
Only the link from the Field name to field id was changed.

![image](https://user-images.githubusercontent.com/1747025/54767502-a3cfd480-4bd3-11e9-9aa0-535d33c3f969.png)